CourseApp.Tests/TestForTask.cs

```diff
@@ -0,0 +1,23 @@
+namespace CourseApp.Tests
+{
+    using CourseApp.Class;
+    using Xunit;
+
+    public class TestForTask
+    {
+        [Theory]
+        [InlineData("VSCode", ".exe", 1.56, "GB")]
+        [InlineData("ISUCT", ".HTML", 0.79, "KB")]
+        [InlineData("Program", ".cs", 0.47, "B")]
+        [InlineData("Music", ".mp3", 1.56, "KB")]
```

Comment:
Are we using `human` size? with 1000 instead of 1024

Reply:
Fixed